src/prop/cadical.cpp

```diff
@@ -1235,7 +1252,25 @@ std::vector<SatLiteral> CadicalSolver::getDecisions() const
 
 std::vector<Node> CadicalSolver::getOrderHeap() const { return {}; }
 
-std::shared_ptr<ProofNode> CadicalSolver::getProof() { return nullptr; }
+std::shared_ptr<ProofNode> CadicalSolver::getProof()
+{
+  Unimplemented() << "getProof for CaDiCaL not supported";
+  return nullptr;
+}
+
+std::pair<ProofRule, std::vector<Node>> CadicalSolver::getProofSketch()
+{
+  Assert(d_logProofs);
+  d_solver->flush_proof_trace();
+  std::vector<Node> args;
+  NodeManager* nm = NodeManager::currentNM();
+  Node pfile = nm->mkConst(String(d_pfFile));
+  args.push_back(pfile);
```

Comment:
This is very verbose for no obvious reason, why not use
```suggestion
  std::vector<Node> args = {NodeManager::currentNM()->mkConst(String(d_pfFile))};
```